sspl_test/coverage/coverage_code

```diff
@@ -6,20 +6,4 @@ co = Coverage(
             # config_file='/tmp/sspl/.coveragerc',
  			)
         logger.info("Starting coverage report scope")
-        co.start()
-    logger.info("Ending Coverage Scope")
-    co.stop()
-    logger.info("coverage object stopped.")
-    co.save()
-    logger.info("coverage info saved.")
-    cov_per = co.xml_report(
- 						outfile='/tmp/sspl/sspl_xml_coverage_report.xml',
- 						ignore_errors=True,
- 					)
-    logger.info(f"XML coverage report generated with coverage of {cov_per}%")
- 	## Enable below code to inable HTML report generation
- 	# html_cov_per = co.html_report(
- 	#                     directory='/tmp/sspl/sspl_html_coverage',
- 	#                     ignore_errors=True,
- 	#                 )
- 	# logger.info(f"HTML coverage report geverated with coverage of {html_cov_per}%")
+        co.start()
```

Comment:
`coverage_code` seems meaningless. Can we rename this something like `sspl_code_coverage.py`?

Reply:
but it is not a python file, it just has the patches of code to be inserted in the sspl_ll_d file.